flink-table/flink-table-runtime/src/test/java/org/apache/flink/table/runtime/operators/join/deltajoin/StreamingDeltaJoinOperatorTest.java

```diff
@@ -679,11 +730,23 @@ public static void release() {
             Objects.requireNonNull(lock).countDown();
         }
 
+        public static void throwException(Throwable t) {
```

Comment:
nit: throwException implied it will throw an Exception. I suggest rename to `setExpectedThrownException `or storeException

Reply:
Good catch!